controllers/disruption_controller.go

```diff
@@ -1027,18 +1028,18 @@ func (r *DisruptionReconciler) generatePod(instance *chaosv1beta1.Disruption, ta
 		}
 	}
 
+	r.InjectorLabels[chaostypes.TargetLabel] = targetName                      // target name label
```

Comment:
Those labels are different depending on the chaos pod related disruption while `r.InjectorLabels` is at the reconciler scope (dealing with all disruptions). I'd prefer to create a new labels map from `r.InjectorLabels` and add disruption specific labels to it instead of modifying the reconciler map directly to avoid any race condition if we decide to enable concurrent reconciles later.

Reply:
This could be related to why the e2e-tests fail?